libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteController.kt

```diff
@@ -25,7 +26,11 @@ internal class FasterRouteController(
         val previousFasterRouteObserver = this.fasterRouteObserver
         this.fasterRouteObserver = fasterRouteObserver
         if (previousFasterRouteObserver == null) {
-            fasterRouteTimer.restartAfterMillis = fasterRouteObserver.restartAfterMillis()
+            val restartAfterMinutes = fasterRouteObserver.restartAfterMinutes()
+            check(restartAfterMinutes in 1..100) {
+                "Faster route should be between 1..100 minutes, $restartAfterMinutes is out of range"
```

Comment:
is this range too conservative? would people want a 300 minute faster route check? maybe < 1000 since 1000 is the first 1000 millisecond mistake?

Reply:
any magic number won't be perfect to all. pick one you like is fine 😉